### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by Mad-Spy**
- Added support for customNames on models
- Added support for higher Specialist levels

### CODE REVIEW

**Comment by DaeronAlagos** [comment]
Thanks for the PR. Can you post the snippet of the points line from the rendered html here please? The cells look a bit odd when there is no name. I want to see how it looks to you and then adjust the styling to it looks standardised on both.

Good catch on the higher levels. It felt like that should have been a loop, but I only had one entry in my sample ros file. Do the levels stack? ie there's no chance of it remaining 7.5cm when a unit is level 4.

**Comment by Mad-Spy** [comment]
Yeah, it does render oddly but I didn't have time to look at the styling. It probably just needs a fixed width putting on the cells.

It's not going to fit nicely on the card for higher levels because you would need to display all of them. Maybe a separate section with a layout styled like the Tactics cards?

File: killteam/data-card.xsl [hunk 1]
```diff
@@ -44,13 +44,21 @@ encoding="UTF-8" indent="yes"/>
...
                         <xsl:variable name="subTotal" select="exslt:node-set($nodePoints)"/>
-                        
-                        <!-- Points -->
-                        <div class="extra" style="text-align:right;">
-                        	<xsl:value-of select="sum($subTotal/ItemCost) + bs:costs/bs:cost/@value"/>
-                        	<!-- <xsl:value-of select="sum(bs:costs/bs:cost/@value)"/> -->
-                        	Points
-                        </div>
+						
+						<div class="extra">
```
Comment (DaeronAlagos) [inline killteam/data-card.xsl:48]: Please replace with

```
<div class="extra">
							<div style="float:left;"><xsl:value-of select="@customName"/></div>
							<div style="float:right;"><xsl:value-of select="sum($subTotal/ItemCost) + bs:costs/bs:cost/@value"/> Points</div>
						</div>
```

**Review by DaeronAlagos — CHANGES_REQUESTED** [review]
See inline comment regarding the customName. I think using divs here looks better. If no name the div collapses, otherwise it aligns left.

**Comment by DaeronAlagos** [comment]
OK, I found the instructions to merge locally (with my suggested changes). When I pushed back to my master it closed the PR. Thanks again for your submission.

**Pull request merged**